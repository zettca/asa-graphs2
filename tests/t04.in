8 11
2
7 3 0
1 6 3
6 4 -3
8 1 1
8 4 0
3 5 -3
2 7 0
4 3 -2
7 1 1
8 2 2
6 1 -2
