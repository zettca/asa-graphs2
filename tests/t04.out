1
0
-1
1
-4
4
0
U
